app/src/main/java/com/kickstarter/viewmodels/RootCommentViewHolderViewModel.kt

```diff
@@ -42,6 +50,17 @@ interface RootCommentViewHolderViewModel {
                 .compose(Transformers.takeWhen(this.onShowCanceledPledgeRootCommentClicked))
                 .compose(bindToLifecycle())
                 .subscribe { this.showCanceledPledgeRootComment.onNext(CommentCardStatus.CANCELED_PLEDGE_COMMENT) }
+
+            commentCardData
+                .withLatestFrom(
+                    Observable.merge(
```

Comment:
I think you can use here instead of 
`Observable.merge(
                        currentUser.loggedInUser(), ---> emmits `user` when there is one, do not emmit if no user
                        currentUser.loggedOutUser() ---> emmits `null` if no user, do not emmit in case there is an user
                    )`
directly 
`currentUser` observable, it will emmit `null` in case the user is logged out and `user` in case there is one logged inin.

Reply:
Thank you for the tip, updated 😄